### PR TITLE
This commit removes all old flair files and adds broken out flair fil…

### PR DESCRIPTION
**Pull request by kalebhermes**
…es. The next commit will have updated styles for these images.

### CODE REVIEW

File: user_flair/general_flair.scss
Comment (yuna9) [inline user_flair/general_flair.scss]: Should be `.flair-g-silenced::before`

File: user_flair/new_hero_flair.scss
Comment (yuna9) [inline user_flair/new_hero_flair.scss]: In my opinion, class names should be kept all lowercase. I don't think we'll run into any collision issues by not capitalizing hero names. What do you think?

File: user_flair/new_hero_master_flair.scss
Comment (yuna9) [inline user_flair/new_hero_master_flair.scss]: We could use the `h-m` prefix or `hm` prefix for these. Shorter class names plus still clear.

File: user_flair/new_hero_master_flair.scss
Comment (yuna9) [inline user_flair/new_hero_master_flair.scss]: A couple extra newlines at the end here.

File: custom.scss
Comment (yuna9) [inline custom.scss]: What's the reason for `-183px` here?

**Review by yuna9 — CHANGES_REQUESTED** [review]
Looks pretty good to me!

File: user_flair/general_flair.scss
Comment (kalebhermes) [reply user_flair/general_flair.scss]: Is bad. Will fix.

File: user_flair/new_hero_flair.scss
Comment (kalebhermes) [reply user_flair/new_hero_flair.scss]: Agreed. I copy - pasted from a big list all at once. I'll lowercase them.

File: user_flair/new_hero_master_flair.scss
Comment (kalebhermes) [reply user_flair/new_hero_master_flair.scss]: Good idea. Will do.

File: user_flair/new_hero_master_flair.scss
Comment (kalebhermes) [reply user_flair/new_hero_master_flair.scss]: :+1:

File: custom.scss
Comment (kalebhermes) [reply custom.scss]: This took me a second to puzzle out. I guess when they did the flair originally, they didn't base it on the 20x40 that normal flair is, so -9px is where the image starts when it's centered in a 40px wide space. The 183 is because that's where it starts vertically.

**Review by yuna9 — APPROVED** [review]

**Comment by yuna9** [comment]
@kalebhermes Before merging, mind squashing those commits together?

**Comment by yuna9** [comment]
If you could, please rebase from master so we don't merge merge commits back into master.

**Pull request merged**
